packages/itwin/tree-widget/src/tree-widget-react/components/trees/classifications-tree/ClassificationsTree.tsx

```diff
@@ -31,13 +36,35 @@ export function ClassificationsTree({
   emptyTreeContent,
   getDecorations,
   getActions,
+  onLabelChanged,
 }: ClassificationsTreeProps) {
   const { categoriesTreeProps, rendererProps } = useClassificationsTree({
     activeView,
     hierarchyConfig,
     emptyTreeContent,
   });
 
+  const nodeActions = useCallback(
+    (node: PresentationHierarchyNode) => {
+      return [<RenameAction key="RenameAction" />, ...(getActions ? getActions(node) : [])];
+    },
+    [getActions],
+  );
+
+  const getEditingProps = useCallback<Required<ComponentProps<typeof StrataKitTreeRenderer>>["getEditingProps"]>(
+    (node) => {
+      return {
+        onLabelChanged: onLabelChanged
+          ? (newLabel) => {
+              node.label = newLabel;
```

Comment:
This does not look right. Component should not directly mutate node. Why is this needed?

Reply:
Forgot to save after experimenting, removed.